doc/rst/usingchapel/chplenv.rst

```diff
@@ -721,26 +726,25 @@ Generating Configuration Files
 To generate a configuration file, use ``printchplenv`` or
 ``./configure``.
 
-When using ``printchplenv``, run it with ``--simple`` or
-``--overrides`` to get a format compatible with Chapel configuration
-files.
+When using ``printchplenv``, run it with the ``--simple`` format flag to get a
+format compatible with Chapel configuration files.
 
-The ``printchplenv --overrides`` flag can be used to print the variables
+The ``--overrides`` filter flag can be used to print only the variables
 currently overridden by either environment variables or Chapel
 configuration file.
 
 For example, to save the current overrides into a Chapel configuration file:
 
 .. code-block:: sh
 
-    printchplenv --overrides > ~/.chplconfig
+    printchplenv --all --simple --overrides > ~/.chplconfig
 
-The ``printchplenv --simple`` flag can be used to print all the variables
+The ``printchplenv --all --simple`` flag can be used to print all the variables
 of the current configuration. For example:
 
 .. code-block:: sh
 
-    printchplenv --simple > ~/.chplconfig
+    printchplenv --all --simple > ~/.chplconfig
```

Comment:
It would be nice if the documentation here referred to printchplenv -h somewhere. I'm happy that printchplenv -h already says its output is compatible with chplconfig format.